include/chemfiles/Property.hpp

```diff
@@ -200,6 +200,8 @@ inline bool operator!=(const Property& lhs, const Property& rhs) {
     return !(lhs == rhs);
 }
 
+using properties_iterator = std::unordered_map<std::string, Property>::const_iterator;
```

Comment:
Should this be define inside the `property_map` class, as `property_map::iterator` ?